base/monit.supervisord.conf

```diff
@@ -0,0 +1,5 @@
+[program:monit]
```

Comment:
We discussed this - we concluded that actually zombie processes were unlikely to be a problem (monit creates temporary defunct processes, because it reaps them on a timer, but they're not orphaned processes which need adoption). You're going to try and figure out what the issues were and let me know - I'm then happy to do any additional testing needed.